libraries/matrix/api/src/main/kotlin/io/element/android/libraries/matrix/api/timeline/item/event/EventTimelineItem.kt

```diff
@@ -46,10 +45,7 @@ data class EventTimelineItem(
     }
 }
 
-fun interface EventDebugInfoProvider {
-    fun get(): TimelineItemDebugInfo
-}
-
-fun interface EventShieldsProvider {
```

Comment:
We could still keep these, wrapping the lazy provider. That way we could also use the lambdas for the fakes.

Reply:
Good point, let me give it a try

Reply:
0db54df877a80fc90ee1d910a9dfee2d60f7553a